### PR TITLE
Fix blurred popovers

### PR DESCRIPTION
**Pull request by savindi7**
Screenshot 2022-09-22 at 21 02 35

> Fix

Screenshot 2022-09-22 at 21 03 51

### Related Issues
- None

### Related PRs
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by DonOmalVindula — APPROVED** [review]

File: modules/react-components/src/components/input/copy-input-field.tsx [hunk 1]
```diff
@@ -55,9 +55,9 @@ export interface CopyInputFieldPropsInterface extends IdentifiableComponentInter
 /**
  * Copy to clipboard input field component.
  *
- * @param {CopyInputFieldPropsInterface} props - Props injected to the component.
+ * @param props - Props injected to the component.
  *
```
Comment (dasuni-30) [inline modules/react-components/src/components/input/copy-input-field.tsx:59]: Can we remove the redundant line?

File: modules/react-components/src/components/input/copy-input-field.tsx [hunk 2]
```diff
@@ -55,9 +55,9 @@ export interface CopyInputFieldPropsInterface extends IdentifiableComponentInter
...
- * @return {React.ReactElement}
+ * @returns React.ReactElement
```
Comment (dasuni-30) [inline modules/react-components/src/components/input/copy-input-field.tsx:60]:
```suggestion
 * @returns Copy Input Field Component.
```

File: modules/react-components/src/components/input/copy-input-field.tsx [hunk 3]
```diff
@@ -91,7 +91,7 @@ export const CopyInputField: FunctionComponent<CopyInputFieldPropsInterface> = (
     /**
      * Copies the value to the users clipboard.
      *
-     * @param event {MouseEvent<HTMLButtonElement>)}
+     * @param event - MouseEvent<HTMLButtonElement>)
```
Comment (dasuni-30) [inline modules/react-components/src/components/input/copy-input-field.tsx:94]:
```suggestion
     * @param event - MouseEvent<HTMLButtonElement>
```

File: modules/react-components/src/components/input/copy-input-field.tsx [hunk 4]
```diff
@@ -113,26 +113,17 @@ export const CopyInputField: FunctionComponent<CopyInputFieldPropsInterface> = (
...
-                            />
-                        )
-                    }
-                    openOnTriggerFocus
-                    closeOnTriggerBlur
-                    position="top center"
-                    content={ copied ? "Copied!" : "Copy to clipboard" }
-                    inverted
+                <Button
+                    data-tooltip={ copied ? "Copied!" : "Copy to clipboard" }
```
Comment (dasuni-30) [inline modules/react-components/src/components/input/copy-input-field.tsx:117]: Can we add i18n strings?

File: modules/react-components/src/components/input/copy-input-field.tsx (hunk 3 above)
Comment (savindi7) [reply modules/react-components/src/components/input/copy-input-field.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/3477/commits/33110800011c556fb355e88506b69eade2e95cef

File: modules/react-components/src/components/input/copy-input-field.tsx (hunk 2 above)
Comment (savindi7) [reply modules/react-components/src/components/input/copy-input-field.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/3477/commits/33110800011c556fb355e88506b69eade2e95cef

File: modules/react-components/src/components/input/copy-input-field.tsx (hunk 1 above)
Comment (savindi7) [reply modules/react-components/src/components/input/copy-input-field.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/3477/commits/33110800011c556fb355e88506b69eade2e95cef

**Review by dasuni-30 — APPROVED** [review]

**Pull request merged**